Sources/TunnelKitWireGuardAppExtension/WireGuardTunnelProvider.swift

```diff
@@ -146,6 +176,27 @@ extension WireGuardTunnelProvider {
         // store path for clients
         cfg._appexSetDebugLogPath()
     }
+
+    func fetchDataCount(completiondHandler: @escaping (Result<WireGuardDataCount, Error>) -> Void) {
+        adapter.getRuntimeConfiguration { configurationString in
+            if let wireGuardDataCount = WireGuardDataCount(from: configurationString) {
+                completiondHandler(.success(wireGuardDataCount))
+            } else {
+                completiondHandler(.failure(StatsError.parseFailure))
+            }
+         }
+    }
+
+    enum StatsError: LocalizedError {
```

Comment:
Maybe, also it's not really localized:

```suggestion
    private enum StatsError: Error {
```

Reply:
Ha, true
Fixed